docs/source/quickstart.rst

```diff
@@ -30,71 +30,102 @@ For example, to run the command on sample data included in ASPIRE (a single ``sa
 
 Use the ``--help`` argument with the command to see the several options associated with this command.
 
-2. Simulation
-#############
+2. Particle Extraction
+######################
 
-The ``simulation`` command simulates a virtual particle made up of multiple gaussian blobs, generates of set of (noisy) images,
-runs the ASPIRE pipeline to determine the estimated mean volume and estimated covariance on the mean volume,
-and runs evaluations on these estimated quantities (against the `true` values which we know from the simulation).
+Given a dataset of full micrographs (``*.mrc`` file) and corresponding coordinate files containing the locations
+of picked particles in the ``*.mrc``, the ``extract-particles`` command extracts the particles into one or more ``.mrcs``
+stacks and generates a ``.star`` file.
+
+Example usage:
+
+.. code-block::
+
+    aspire extract-particles --mrc_paths=my/data/sample.mrc --coord_paths=my/data/coords/sample.coord --starfile_out=my_dataset_stack.star --particle_size=256 --centers
+
+3. Estimate Contrast Transfer Function
+######################################
+
+The ``estimate-ctf`` command estimates the CTF from experimental data and returns the CTF as a mrc file.  For example,
 
 .. code-block:: console
 
-    aspire simulation
+      python -m aspire estimate-ctf --data_folder path_to_input_data_folder
```

Comment:
Why are we doing `python -m` here but not above

Reply:
Not sure. It was already there. I don't think it's necessary. removed.

Reply:
Its just legacy, oversight.  I probably was the one that added the `aspire` entrypoint and didn't get all the docs.